src/xpcc/communication/xpcc/backend/zeromq/reader.hpp

```diff
@@ -0,0 +1,177 @@
+// coding: utf-8
+/* Copyright (c) 2016, Roboterclub Aachen e. V.
```

Comment:
@chris-durand this needs to be the actual person who created this code, German copyright law afaik does not allow you to assign copyright to a company